code/game/machinery/computer/aifixer.dm

```diff
@@ -148,3 +148,7 @@
 			to_chat(user, "<span class='boldannounce'>ERROR</span>: Reconstruction in progress.")
 		else if (!occupier)
 			to_chat(user, "<span class='boldannounce'>ERROR</span>: Unable to locate artificial intelligence.")
+
+obj/machinery/computer/aifixer/on_deconstruction()
+	if(occupier)
+		occupier.Destroy()
```

Comment:
you might want to null out occupier after this?

Reply:
`Destroy` is the wrong call to use here. In general it should be `qdel(occupier)` to delete something, and in this case to set it to null afterwards, `QDEL_NULL(occupier)`